lineman/app/models/base_model.coffee

```diff
@@ -27,7 +27,10 @@ angular.module('loomioApp').factory 'BaseModel', ->
       @updateFromJSON(data)
 
     clone: ->
-      attrs = @baseSerialize()[@constructor.singular] or {}
+      attrs = _.reduce @constructor.attributeNames, (clone, attr) =>
```

Comment:
We already store the attribute names we import via the constructor, and using that as the base of things to clone works much better than attempting to serialize.